src/Chapter1.hs

```diff
@@ -614,9 +623,11 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
-
+sumLast2 :: Int -> Int
+sumLast2 n = lastDigit (abs n)  + lastDigit (div (abs n) 10)
+  where
+    lastDigit :: Int -> Int
+    lastDigit n = let remainder = mod n 10 in remainder
```

Comment:
Great one 👌🏼 

Some additional info to this one: in the standard library, there is the `divMod` function, that is doing what the `lastDigit` here 🙂